en/admin/installation/installation.md

```diff
@@ -9,6 +9,12 @@ git clone https://github.com/wallabag/wallabag.git
 cd wallabag && make install
 ```
 
+If you want to use LDAP, replace last command with
+```bash
+cd wallabag && make install LDAP_ENABLED=true
+```
```

Comment:
@nicosomb is that a problem? when the feature requires a "big" change (here, adding the ldap extension to php), it’s always good to be able to enable it only "on demand", isn't it?